### PR TITLE
MUWM-5272

### PR DESCRIPTION
**Pull request by fanglinfang**
https://jira.cac.washington.edu/browse/MUWM-5272

### CODE REVIEW

File: myuw_vue/components/textbooks/textbooks.vue [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <template>
   <uw-panel :loaded="isReady" :errored="isErrored">
     <template #panel-body>
+      <div
+        v-if="student && !tacoma"
+        class="alert myuw-text-md" role="alert"
+      >
+        UW Day One Access Program
```
Comment (fanglinfang) [inline myuw_vue/components/textbooks/textbooks.vue:8]: William, please add the textbook static content here.

File: myuw_vue/components/accounts/tuition-fees.vue [hunk 2]
```diff
@@ -130,6 +130,55 @@
             </template>
           </uw-card-status>
         </li>
+
+        <li v-if="hasIacData">
+          <h3>UW Day One Access Fees</h3>
+          <div>
+            One of ...
```
Comment (fanglinfang) [inline myuw_vue/components/accounts/tuition-fees.vue:137]: William, please add the static content.

File: docker/settings.py [hunk 3]
```diff
@@ -207,9 +207,11 @@
     }
 }
 
-DEBUG = False
+DEBUG = True
```
Comment (devights) [inline docker/settings.py:210]: Is this a permanent change? Any exceptions will include most of the Django settings and other configuration in the output to the user.  

Also, if debug is False ALLOWED_HOSTS must be set or all requests will fail: https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-DEBUG

Comment (fanglinfang) [reply docker/settings.py:210]: No, not permanent .

**Review by devights — APPROVED** [review]

**Pull request merged**